### PR TITLE
[Refactor]: UI Updates

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
This PR covers: 

- Time selection and Stage and run in the same row
- remove old add to dashboard and setup alerts buttons - logs explorer
- Fix the tooltip issue - select view
- Clear selected view option

Loom: https://www.loom.com/share/d9d683afec96451781e85d81bed4e4f9?sid=42130a0f-13fc-454a-ae9d-089b35a0efe4

### CODE REVIEW

File: frontend/src/container/ExplorerOptions/ExplorerOptions.styles.scss [hunk 1]
```diff
@@ -41,6 +41,11 @@
 
 				font-weight: 500;
 			}
+
+			&.ant-btn-round:disabled {
+				background-color: rgba(209, 209, 209, 0.074);
+				color: #5f5f5f;
```
Comment (YounixM) [inline frontend/src/container/ExplorerOptions/ExplorerOptions.styles.scss:47]: Use variables, Handle light mode

File: frontend/src/container/ExplorerOptions/ExplorerOptions.tsx [hunk 2]
```diff
@@ -38,48 +56,140 @@ function ExplorerOptions({
...
+				compositeQuery: mapCompositeQueryFromQuery(currentQuery, panelType),
+				viewKey,
+				extraData: '',
+				sourcePage: sourcepage,
+				viewName,
+			},
+			{
+				onSuccess: () => {
+					notifications.success({
+						message: 'View Updated Successfully',
```
Comment (YounixM) [inline frontend/src/container/ExplorerOptions/ExplorerOptions.tsx:98]: use strings constant

**Review by YounixM — APPROVED** [review]

**Review by Vikrant2520 — APPROVED** [review]

File: frontend/src/container/ExplorerOptions/ExplorerOptions.styles.scss (hunk 1 above)
Comment (Rajat-Dabade) [reply frontend/src/container/ExplorerOptions/ExplorerOptions.styles.scss:47]: This looks good for both dark and light mode.

File: frontend/src/container/ExplorerOptions/ExplorerOptions.tsx (hunk 2 above)
Comment (Rajat-Dabade) [reply frontend/src/container/ExplorerOptions/ExplorerOptions.tsx:98]: Sure, will take care of it in subsequent PR.

**Pull request merged**